neuromation/cli/formatters/jobs.py

```diff
@@ -90,7 +90,7 @@ def __call__(self, job_status: JobDescription) -> str:
         result += f"Preemptible: {job_status.is_preemptible}\n"
         if job_status.internal_hostname:
             result += f"Internal Hostname: {job_status.internal_hostname}\n"
-        http_url = job_status.http_url_named or job_status.http_url
+        http_url = job_status.http_url
```

Comment:
I think it can be inlined, as `job_status.internal_hostname` or other attributes.

Reply:
done